documentation/usage/installation.md

```diff
@@ -2,18 +2,153 @@
+
+That's everything! At this point you should have a working copy of the game to play. Join us in the [discord](https://discord.gg/VZbXMHXzWv) or finding streams/streaming yourself on [Twitch](https://www.twitch.tv/directory/game/Jak%20and%20Daxter:%20The%20Precursor%20Legacy) to get involved in the community.
+
+**ENJOY!**
 
 ## Linux
 
-TODO
+The instructions for Linux users are a bit more sparse than the Windows users, but most of the steps are identical.
+
+To begin with you'll first need to create a `.iso` file from your copy of Jak and Daxter, reading the dics for its content. The [Windows ISO instructions](installation#creating-your-iso) have some example instructions using a Windows program called ImgBurn, which you'll need an alternative to, but the rest of the instructions are the same.
+
+The biggest change is how you run OpenGOAL, as we don't provide a pre-built Linux installer, so the instructions for building the game are below.
+
+### Running OpenGOAL on Linux
+
+As there is no pre-built launcher for Linux you will need to instead fetch the latest release from our [GitHub repo](https://github.com/open-goal/jak-project/releases). Here you can choose either the `.bin` installer or extract the files yourself from the `.tar.gz` folder to get the tools for this setup.
+
+If you have some technical experience, and would prefer to build this application from the source yourself, then take a look at the [project README instructions](https://github.com/open-goal/jak-project#setting-up-a-development-environment) instead. Support for this however is outside the scope of this document.
+
+If you're continuing with the builds from the latest releases, either install the `.bin` or extract the `.tar.gz` content to get an install with a similar list of files to the below.
+
+![Expected files for Linux users](/docs/installation/linux_files_list.png)
+
+Once you have these files extracted, there are two main steps required. The first is to extract all of the assets from the `.iso` file you created earlier, using the `extractor` from the install above (eg `./extractor <path-to-iso>`). Once this extraction completes, you should be able to run `gk` from the same folder to start the game.
+
+That's everything! At this point you should have a working copy of the game to play. Join us in the [discord](https://discord.gg/VZbXMHXzWv) or finding streams/streaming yourself on [Twitch](https://www.twitch.tv/directory/game/Jak%20and%20Daxter:%20The%20Precursor%20Legacy) to get involved in the community.
 
 ## MacOS
 
 :::danger
-We do not support MacOS at this time
+We do not officially support MacOS at this time. You may have some success with the [Linux installation instructions](installation#linux), but we will not provide any support if this does not work.
```

Comment:
They will have no success!  macOS doesn't support the version of OpenGL we use, and if they are on an Apple Silicon device, we don't support Rosetta/ARM.  There are several people that want to support macOS in the future but yeah, currently unsupported

Reply:
Ah that's a shame, I've gotten used to things built for linux often just working on macs, but I suppose most of those aren't really GUI tools :P I've changed this note to

`We do not support macOS at this time. Unfortunately macOS does not currently support the version of OpenGL we use for this project, and we additionally do not support any ARM processors, meaning significant changes would be required.`

for the next commit as I think that's good info to have, but I'm happy to just revert back to the original unsupported message if that's preferrable